### DISCUSSION TITLE
Exporting passwords used in the infra

### DISCUSSION THREAD

**Discussion by JanoutV** [telekom-security/tpotce#1092] [Q&A]
Hello there, I would like to create a wordlist from password which were used on my T-Pot infra. Does anybody have an idea on how I could export all of them in the easiest way possible? I like how the Kibana dashboard shows the most used passwords, as on the picture
![image](https://user-images.githubusercontent.com/44194415/168079294-0ad48ba1-64b6-432c-8ac8-aa8c75d2636b.png)
but I'd like to get all of the passwords used into a single wordlist file.

**Comment by t3chn0m4g3** [comment]
You find example queries in the discussions and in `/opt/tpot/bin/`.

**Comment by lcia-projects** [comment]
i've got a python script that does this. it builds a text file and a csv with the results.
it was kinda custom to what i was doing, but i can post the code if you'd like. 

i basically used python + elasticsearch_dsl library. made one query for all the usernames in the last hour, and another for the passwords in the last hour. it pulled the data, consolidated it.. then wrote a csv and a text file of each.

**Reply by JanoutV** [reply]
If you could post it so I could use it as a headstart that would be great!
If I make any modifications I could share those back.
